bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -20,3 +20,12 @@ data:
     [gateway]
     mode=local
     nodeport=true
+
+{{ if .OVNHybridOverlayEnable }}
+    [hybridoverlay]
+    enabled=true
+    cluster-subnets="{{.OVNHybridOverlayNetCIDR}}"
```

Comment:
To be consistent with the bash, you need `{{if .OVNHybridOverlayNetCIDR}}`...`{{end}}` around this. I think that case is used with the non-Windows overlay mode.

Reply:
I am not sure you are right 
```
          hybrid_overlay_flags=
          if [[ -n "{{.OVNHybridOverlayEnable }}" ]]; then
            hybrid_overlay_flags="--enable-hybrid-overlay --no-hostsubnet-nodes=kubernetes.io/os=windows"
            if [[ -n "{{.OVNHybridOverlayNetCIDR}}" ]]; then
              hybrid_overlay_flags="${hybrid_overlay_flags} --hybrid-overlay-cluster-subnets={{.OVNHybridOverlayNetCIDR}}"
            fi
            if [[ -n "{{.OVNHybridOverlayVXLANPort}}" ]]; then
              hybrid_overlay_flags="${hybrid_overlay_flags} --hybrid-overlay-vxlan-port={{.OVNHybridOverlayVXLANPort}}"
            fi
          fi```

is  the original bash so in order to define a HybridOverlayNetCIDR OVNHybridOverlay has to be enabled